From: Zhong Jianxin <azuwis@gmail.com>
Date: Fri, 14 Aug 2015 09:51:20 +0800
Subject: Fix zsh completion

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index d0f0d9f..17acde3 100644
--- a/Makefile
+++ b/Makefile
@@ -10,7 +10,7 @@ PREFIX   ?= /usr/local
 BINPREFIX = $(PREFIX)/bin
 MANPREFIX = $(PREFIX)/share/man
 BASHCPL = $(PREFIX)/share/bash-completion/completions
-ZSHCPL = $(PREFIX)/share/zsh/site-functions
+ZSHCPL = $(PREFIX)/share/zsh/vendor-completions
 DOCPREFIX = $(PREFIX)/share/doc/bspwm
 
 MD_DOCS = doc/CONTRIBUTING.md doc/INSTALL.md doc/MISC.md doc/TODO.md
